ext/tags.go

```diff
@@ -108,7 +108,7 @@ func (r rpcServerOption) Apply(o *opentracing.StartSpanOptions) {
 	if r.clientContext != nil {
 		opentracing.ChildOf(r.clientContext).Apply(o)
 	}
-	(opentracing.Tags{string(SpanKind): SpanKindRPCServer}).Apply(o)
+	opentracing.Tag{string(SpanKind), SpanKindRPCServer}.Apply(o)
```

Comment:
thinking aloud, we could make `SpanKindRPCServer` a `Tag` proper and allows callers to deref `SpanKindRPCServer.Key` and `.Value` as they wish...